deploy/vagrant/provision/roles/spark/files/remove_s3_exclusion.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+python <<EOF
+import re
+
+PATTERN = r'''<exclusion>
+\s*<groupId>org\.tachyonproject</groupId>
+\s*<artifactId>tachyon-underfs-s3</artifactId>
+\s*</exclusion>'''
+POM = '/spark/core/pom.xml'
+
+new_pom = re.compile(PATTERN).sub('', open(POM).read())
+open(POM, 'w').write(new_pom)
+EOF
```

Comment:
Can we just have this as a py file and run this py file in the config.yml, rather than encoding python content in a .sh file?